features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -75,6 +75,10 @@ const Logo = styled.img`
   @media (min-width: ${breakpoint("desktop")}) {
     margin: ${space(0, 4)};
   }
+
+  @media (max-width: 1023px) {
```

Comment:
This is basically the desktop breakpoint so it would be great to reuse it. You can do so by using the CSS `calc` function like below. Haven't tested it but should work 🙂 

```
@media (max-width: calc(${breakpoint("desktop")} - 1px) { ... }
```

Reply:
Oh yeah! haven't thought about this.... I 've never used calc fun before bec I don't know how to use it lol! thanks!!